preview/_/js/previm.js.tmpl

```diff
@@ -11,6 +11,30 @@
                    .use(_win.markdownitCheckbox)
                    .use(_win.markdownitCjkBreaks);
 
+  var defaultRenderer =
+    md.renderer.rules.link_open ||
+    function (tokens, idx, options, env, self) {
+      return self.renderToken(tokens, idx, options);
+    };
+
+  md.renderer.rules.link_open = function (tokens, idx, options, env, self) {
+    var targetIndex   = tokens[idx].attrIndex("target");
+    if (targetIndex < 0) {
+      if (!/^#/.test(tokens[idx].attrs[0][1])) {
+        tokens[idx].attrPush(["target", "_blank"]);
+
+        var relIndex  = tokens[idx].attrIndex("rel");
+        if (relIndex < 0) {
+          tokens[idx].attrPush(["rel", "noopener"]);
+        } else {
+          tokens[idx].attrs[relIndex][1] = "noopener";
+        }
```

Comment:
:question:

Is this code overwrite user specified `rel` attriute to `"noopener"`? (I'm not sure how users can specify `rel` attribute in Markdown though...)

Reply:
Yes, it does overwrite the existing `rel` attribute.
I thought that Markdown syntax cannot specify link attributes in detail, including `rel`.
It would never come to this line.

Reply:
I see. Well, I don't remember if previm allowed it but (some) Markdown allows raw HTML so still user can write a raw `<a>` tag with `rel` attribute I guess and I think we should not overwrite it if user specified it. What do you think?

Reply:
By checking `tokens[idx]`, it looks `<a>` tags don't come here.
It's enough to care about Markdown syntax only.